vesting/src/lib.rs

```diff
@@ -248,6 +255,17 @@ pub mod module {
 			Ok(())
 		}
 
+		/// Verify input by `validate_unsigned
+		#[pallet::weight(T::WeightInfo::claim((<T as Config>::MaxVestingSchedules::get() / 2) as u32))]
+		pub fn claim_for(origin: OriginFor<T>, dest: <T::Lookup as StaticLookup>::Source) -> DispatchResult {
+			ensure_none(origin)?;
```

Comment:
no need to make this unsigned tx.
this is useful to claim rewards for cold wallets that using custom signing software so they don't need to support `vesting.claim`. instead, another account can trigger claim and pay for tx fee.